src/systems/systems/src/hydraulic/linear_actuator.rs

```diff
@@ -47,6 +50,16 @@ pub enum LinearActuatorMode {
     ActiveDamping,
     ClosedCircuitDamping,
 }
+impl Display for LinearActuatorMode {
+    fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
+        match self {
+            LinearActuatorMode::ClosedValves => write!(f, "ClosedValves"),
+            LinearActuatorMode::PositionControl => write!(f, "PositionControl"),
+            LinearActuatorMode::ActiveDamping => write!(f, "ActiveDamping"),
+            LinearActuatorMode::ClosedCircuitDamping => write!(f, "ClosedCircuitDamping"),
+        }
+    }
+}
```

Comment:
**thought**: Seems to be unused; implementation seems to be equal to deriving from `Debug`.
**suggestion**: Remove this implementation. Derive from `Debug` if you need `println!` capabilities. Note that `Display` is for user-facing output.

Reply:
Only println! needed there indeed.

Reply:
I'd like to note that adding `Debug` as part of `#[derive(PartialEq, Clone, Copy, Debug)]` will result in the same output as your current custom implementation. Probably you want to remove it?

Reply:
Always forget about that macro sorcery ..... Thanks!